modules/ROOT/pages/documentation-openapi.adoc

```diff
@@ -116,28 +117,30 @@ See the xref:reference:javadoc/microprofile-3.3-javadoc.adoc#package=org/eclipse
 
 An alternative approach is to design the REST API in an editor, such as the link:https://editor.swagger.io/[Swagger editor], before you write any code.
 With this approach, you can spot and rectify any issues in the design before it is implemented.
-In large companies, APIs need to be consistent and usable, which can be reviewed by subject matter experts to verify the quality of the API design.
+In large companies, subject matter experts review the API to ensure it's consistent and usable.
 This API design then forms a contract and must be implemented as agreed.
 
 You can write this API design in YAML or JSON format and place it in the `META-INF` directory of your application.
 Optionally, you can create stubs for the API code.
-You can augment the documentation with annotations in the code as described.
+The code first and design-first approaches are not mutually exclusive.
+You can augment manually created API documents by adding annotations to the code as you would in a code-first approach.
 
 For more information, see https://openliberty.io/guides/microprofile-openapi.html#using-pregenerated-openapi-documents[Using pregenerated OpenAPI documents].
 
 == Filter components in and out of API documentation
 
 Filters give the users the flexibility to modify the document.
-With https://openliberty.io/guides/microprofile-openapi.html#filtering-the-openapi-tree-elements[filters], you can implement an interface in your application.
+With filters, you can implement an interface in your application.
```

Comment:
looking at this again, I think "For more information" sentence should be at the end of this section. And I'd remove "So" from the second sentence.